### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dilipchauhan14**
updated README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,15 @@ MNO
...
+API is the acronym for Application Programming Interface, 
+which is a software intermediary that allows two applications to talk to each other. 
+Each time you use an app like Facebook, send an instant message, 
+or check the weather on your phone, you’re using an API.
+When you use an application on your mobile phone, 
+the application connects to the Internet and sends data to a server. 
+The server then retrieves that data, interprets it,
+performs the necessary actions and sends it back to your phone. 
+The application then interprets that data and presents you with 
+the information you wanted in a readable way.
```
Comment (Dilipchauhan1998) [inline README.md]: good job

File: README.md [hunk 2]
```diff
@@ -19,3 +19,15 @@ MNO
 123
 456
 789
+API is the acronym for Application Programming Interface, 
+which is a software intermediary that allows two applications to talk to each other.
```
Comment (Dilipchauhan1998) [inline README.md]: bhut badhiya

Comment (dilipchauhan14) [reply README.md]: yes

File: README.md [hunk 3]
```diff
@@ -19,3 +19,15 @@ MNO
 123
 456
 789
+API is the acronym for Application Programming Interface,
```
Comment (dilipchauhan14) [inline README.md]: GOOD WORK

**Pull request merged**